pubspec.yaml

```diff
@@ -145,6 +145,7 @@ flutter:
     - assets/svg/contact_icons/x.svg
     - assets/svg/contact_icons/tiktok.svg
     - assets/svg/contact_icons/discord.svg
+    - assets/svg/contact_icons/storey.svg
```

Comment:
tbh, I think we should have separate digital_guide catalog for our assets related to this.

Reply:
I agree but it's for another task probably

Reply:
And I used ContactSection twice not to duplicate code with slight modifications. I am not sure whether it would be better to remove modifications of ContactSection and create custom section for this. WDYT?

Reply:
Yeah, I think the modifications are very small, so they can stay. I just only meant to move `storey.svg` oto `/aseets/svg/digital_guide`

Reply:
@24bartixx